recipes/beyla/collector-config.yaml

```diff
@@ -74,11 +79,33 @@ spec:
         metrics:
           metric:
             - resource.attributes["k8s.pod.name"] == nil
+      transform/prometheusconventions:
+        metric_statements:
+          - context: metric
+            statements:
+              - set(name, "http_request_duration") where name == "http.server.duration"
+              - set(name, "http_requests") where name == "http.server.calls"
```

Comment:
Is there documentation for these conventions?

Reply:
I was following https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/server/http.  I don't think prometheus itself documents them.

Reply:
added a comment